tools/processTypes.mjs

```diff
@@ -0,0 +1,32 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+
```

Comment:
Can we add a comment explaining what this does and why? I think I get it, but we may forget or a new person might not have a clue.